lib/analyzer/index.ts

```diff
@@ -1,4 +1,3 @@
-import { analyze as analyzeDynamically } from "./docker-analyzer";
 import { analyze as analyzeStatically } from "./static-analyzer";
 
-export { analyzeDynamically, analyzeStatically };
+export { analyzeStatically };
```

Comment:
Since we're getting rid of dynamic scanning it would be really great to do some renaming here, but it's just a note and I appreciate it could be done as a separate PR when we modularise/restructure `sdp` for example, like you mentioned before

Reply:
Agreed! But some tests call this directly and I didn't want to touch too many things while in the middle of refactoring the tests, I'll definitely note it down for a future clean up though!

Reply:
Makes total sense, I thought as much :-) 👍